docs/reference/api-connectors-elasticsearch.md

```diff
@@ -92,6 +92,8 @@ The `ApiProxyConnector` is used when you want to proxy search requests through y
 It sends `onSearch` and `onAutocomplete` requests to your API, which is expected to forward them to Elasticsearch using `ElasticsearchAPIConnector`.
 
 ```js
+import { ApiProxyConnector } from "@elastic/search-ui-elasticsearch-connector/api-proxy";
+// OR
 import { ApiProxyConnector } from "@elastic/search-ui-elasticsearch-connector";
```

Comment:
nit: should we combine this line with OR comment? 

```
// OR 
// import { ApiProxyConnector } from "@elastic/search-ui-elasticsearch-connector";
```

Reply:
Yeah, I agree, hadn't strong opinion about that, but seems like it makes more sense